src/features/DisplayControls/DrugChart/components/DrugChartView.jsx

```diff
@@ -138,6 +138,51 @@ export default function DrugChartWrapper(props) {
     callFetchMedications(startDateTime, endDateTime);
   };
 
+  const shiftTiming = () => {
+    let shiftStartDateTime = formatDate(
+      startEndDates.startDate,
+      dateFormatString
+    );
+    let shiftEndDateTime = formatDate(
+      startEndDates.endDate - 60,
+      dateFormatString
+    );
+    const [shiftStartDate, shiftStartTime] = shiftStartDateTime.split(" | ");
```

Comment:
This could break if we have 12 hrs format as there is no "|" separater in 12 hr date time format